theme/errorList.less

```diff
@@ -1,7 +1,7 @@
 @total-height: 16em;
 @header-height: 4em;
 @inner-height: (@total-height - @header-height);
-@collapsed-errors-hover-color: rgba(0, 0, 0, 0.02);
+@collapsed-errors-hover-color: rgba(0, 0, 0, 0.05);
```

Comment:
Move all LESS variables to https://github.com/microsoft/pxt/blob/master/theme/themes/pxt/globals/site.variables